app/controllers/service_requests_controller.rb

```diff
@@ -306,18 +306,15 @@ def confirmation
   end
 
   def send_confirmation_notifications to_notify
-    if @service_request.previous_submitted_at.nil?
-      send_notifications(@service_request, @sub_service_request)
-    elsif @sub_service_request
-      if to_notify.include? @sub_service_request.id
-        send_ssr_service_provider_notifications(@service_request, @sub_service_request)
-      end
+    if @sub_service_request
+      send_notifications(service_request, [@sub_service_request]) if to_notify.include? @sub_service_request.id
     else
+      sub_service_requests = []
```

Comment:
Is `to_notify` an array? Then this may be easier: 
`sub_service_requests = @service_request.sub_service_requests.where(id: to_notify)`